src/ApiKitClient.ts

```diff
@@ -28,9 +36,13 @@ export class ApiKitClient {
     });
 
     this.instance.interceptors.response.use((response: AxiosResponse) => response, (error: AxiosError) => {
-      if (error.response?.status === 401 && this.unauthorizationCallback) {
+      const { status } = error.response;
+      if (status === 401 && this.unauthorizationCallback) {
         this.unauthorizationCallback();
       }
+      if (this.globalErrorCallback) {
+        this.globalErrorCallback();
```

Comment:
you might need to pass the error object

Reply:
Didn't get passing error object where...?